_gcode/M115.md

```diff
@@ -13,20 +13,49 @@ codes: [ M115 ]
 This command causes Marlin to output a string like this:
 
 ```
-FIRMWARE_NAME:Marlin 1.1.0 (Github) SOURCE_CODE_URL:https://github.com/MarlinFirmware/Marlin PROTOCOL_VERSION:1.0 MACHINE_TYPE:RepRap EXTRUDER_COUNT:1 UUID:cede2a2f-41a2-4748-9b12-c55c62f367ff
+FIRMWARE_NAME:Marlin 2.1.3 (Aug  1 2024 12:00:00) SOURCE_CODE_URL:github.com/MarlinFirmware/Marlin PROTOCOL_VERSION:1.0 MACHINE_TYPE:3D Printer KINEMATICS:Cartesian EXTRUDER_COUNT:1 UUID:cede2a2f-41a2-4748-9b12-c55c62f367ff
```

Comment:
The extra space in the date looked odd, but apparently `__DATE__` is a fixed length, so a space is used when the day is <10:

> `__DATE__` The compilation date of the current source file. The date is a constant length string literal of the form _Mmm dd yyyy_. The month name _Mmm_ is the same as the abbreviated month name generated by the C Runtime Library (CRT) asctime function. The first character of date _dd_ is a space if the value is less than 10. This macro is always defined.

[Source](https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170#standard-predefined-macros)